runtime/jcl/common/mgmtthread.c

```diff
@@ -1222,6 +1222,7 @@ initIDCache(JNIEnv *env)
 	}
 	if (!(gcls = (*env)->NewGlobalRef(env, cls))) {
 		err = JNI_ENOMEM;
+                (*env)->DeleteLocalRef(env, cls);
```

Comment:
Please indent with tabs here, like the rest of the file.

Reply:
done.